src/OpenLoco/src/Vehicles/VehicleBody.cpp

```diff
@@ -1327,4 +1328,13 @@ namespace OpenLoco::Vehicles
             invalidateSprite();
         }
     }
+
+    // 0x0042F6B6
+    // TODO: move this?
+    uint32_t getNumUnitsForCargo(uint32_t maxPrimaryCargo, uint8_t primaryCargoId, uint8_t newCargoId)
```

Comment:
Not sure if this is the right place for this function. I needed to move it out of the vehicle window, though -- it is called in three more locations.

Reply:
shouldn't we just keep the definition here and keep the body in the cpp?

Reply:
Its really a cargo function i guess. Not sure where would be best

Reply:
Maybe move it to be part of the `CargoObject`? Also not ideal, considering it's working with two objects... Perhaps we should just leave it here, for now?

Reply:
Yeah leave for now